data/match/src/commonMain/kotlin/com/adammcneilly/pocketleague/data/match/OctaneGGMatchService.kt

```diff
@@ -25,18 +24,21 @@ class OctaneGGMatchService(
 
     override suspend fun getMatchDetail(matchId: Match.Id): Result<Match> {
         return apiClient.getResponse<OctaneGGMatch>(
-            endpoint = "$MATCHES_ENDPOINT/$matchId",
+            endpoint = "$MATCHES_ENDPOINT/${matchId.id}",
```

Comment:
Test this service, too.

Reply:
This is tricky because this service uses params and the test isn't setup to consider that yet, will revisit.